src/command/output.rs

```diff
@@ -1,11 +1,13 @@
 use std::fmt::Debug;
 use std::{collections::HashMap, fmt::Display};
 
-use ansi_term::Colour::Yellow;
+use crate::utils::table::{self, cell, row};
+use ansi_term::Colour::{Cyan, Yellow};
 use atty::Stream;
+use crossterm::style::Attribute::*;
```

Comment:
nit: i usually like to avoid `*` wherever possible just so it's very clear what's being imported - won't enforce this though if it's helpful shorthand (though it looks like here we may only need `Underlined`?